packages/cli/src/utils/startupWarnings.ts

```diff
@@ -12,12 +12,20 @@ import { getErrorMessage } from '@google/gemini-cli-core';
 const warningsFilePath = pathJoin(os.tmpdir(), 'gemini-cli-warnings.txt');
 
 export async function getStartupWarnings(): Promise<string[]> {
+  const warnings: string[] = [];
```

Comment:
this is also unrelated. please revert or move to a separate pull request. I believe someone else has already sent out a pull request for this as well.